azure-pipelines.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   dependsOn: []
   timeoutInMinutes: 120
   pool:
-    vmImage: 'ubuntu-18.04'
+    vmImage: 'ubuntu-20.04'
   steps:
```

Comment:
Just as a footnote, do we break Ubuntu 18.04-LTS company? That is an Ubuntu community supported release for a 5 year period. My brief checks suggested that Ubuntu 22.04 would remain compatible for Ubuntu 18.04, but that was testing C code and not C++

Reply:
We statically links all libraries except libc, so if C code doesn't break then we shouldn't break. Officially we only provide Docker image for now so that shouldn't break things either.